regress/pesterTests/SCP.Tests.ps1

```diff
@@ -92,8 +115,8 @@ Describe "Tests for scp command" -Tags "CI" {
         # for the first time, delete the existing log files.
         if ($OpenSSHTestInfo['DebugMode'])
```

Comment:
Since we are copying logs.. we can delete DebugMode all together?

Reply:
actually we are not always copy log. Also wehn DebugMode is true, we save both sourcedir and destinationdir

Reply:
can you do the similar change in other files like sftp tests.

Reply:
updated sftp and cfginclude tests